Configure

```diff
@@ -1193,6 +1202,10 @@ $config{build_file} = env('BUILDFILE') || $target{build_file} || "Makefile";
 my %skipdir = ();
 my %disabled_info = ();         # For configdata.pm
 foreach my $what (sort keys %disabled) {
+    # There are deprecated disablables that translate to themselves.
+    # They cause disabling cascades, but should otherwise not regiter.
```

Comment:
register ?

Reply:
Already merged...  but hey, PRs welcome!  :wink:

Reply:
did you mean register or "should otherwise not be of interest?"

Reply:
"register"

Reply:
probably worth a CHANGES entry if you're at it